lib/Doctrine/ORM/Mapping/Driver/ReflectionBasedDriver.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Mapping\Driver;
+
+use Doctrine\ORM\Mapping\ClassMetadata;
+use ReflectionProperty;
+
+trait ReflectionBasedDriver
```

Comment:
Please flag this trait as `/** @internal */`. We probably want to inline it into the attribute driver in 3.0.x.

Reply:
will do